newsfragments/43102.significant.rst

```diff
@@ -0,0 +1,5 @@
+Change in update model endpoints (PATCH) ``update_mask``
+
+The ``update_mask`` parameter in the PATCH endpoints has been updated to explode (true) by default.
+Before: ``http://<URL>:<PORT>/<PATH>?update_mask=item1,item2``
```

Comment:
I don't think this should be specific to the `PATCH` endpoint. This is how fast API works,  and all list parameters in the API will be impacted.

Reply:
Good point! I originally focused on this as a specific case for `update_mask`, which is why it read more like a deployment guide. I agree, this should be written from a broader perspective. Thanks for the feedback! I updated accordingly, I hope the new one works out!